### PR TITLE
refactor(finalizer): Relocate Arbitrum One outbox ABI

### PR DESCRIPTION
**Pull request by pxrl**

### CODE REVIEW

File: src/finalizer/utils/arbitrum.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export async function multicallArbitrumFinalizations(
 export async function finalizeArbitrum(message: L2ToL1MessageWriter): Promise<Multicall2Call> {
   const l2Provider = getCachedProvider(CHAIN_ID, true);
   const proof = await message.getOutboxProof(l2Provider);
+  const outboxAbi = CONTRACT_ADDRESSES[42161].outbox.abi;
   const outbox = new Contract((await getL2Network(l2Provider)).ethBridge.outbox, outboxAbi);
```
Comment (pxrl) [inline src/finalizer/utils/arbitrum.ts:45]: @nicholaspai We grab the outbox contract address from the SDK in this case. wdyt about sourcing it from ContractAddresses as well?

Comment (nicholaspai) [reply src/finalizer/utils/arbitrum.ts:45]: on board with that +1

**Review by nicholaspai — APPROVED** [review]

**Review by james-a-morris — APPROVED** [review]
Good refactor

File: src/common/ContractAddresses.ts [hunk 2]
```diff
@@ -677,5 +677,27 @@ export const CONTRACT_ADDRESSES: {
         },
       ],
     },
+    outbox: {
+      address: "0x0b9857ae2d4a3dbe74ffe1d7df045bb7f96e4840",
```
Comment (pxrl) [inline src/common/ContractAddresses.ts:681]: Address sourced from: https://docs.arbitrum.io/for-devs/useful-addresses#cross-chain-messaging-contracts

Comment (pxrl) [reply src/common/ContractAddresses.ts:681]: Also seen in an existing withdrawal transaction - see event idx 129 in the following transaction: https://etherscan.io/tx/0x916fa895873f78ac73625e4c9aae78a4f2fc9653d2e8ba4e13aa92c813106ade#eventlog.

**Pull request merged**